Okta.AspNet/LoginMode.cs

```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="LoginMode.cs" company="Okta, Inc">
+// Copyright (c) 2018-present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace Okta.AspNet
+{
+     /// <summary>
+     /// LoginMode controls the behavior of the middleware.
```

Comment:
I would elaborate here, maybe something like "LoginMode controls the login redirect behavior of the middleware"